roles/seldon_core_analytics/tasks/main.yaml

```diff
@@ -15,11 +15,21 @@
   register: istio_web_service
   when: seldon_core_analytics_istio_external_ip_required | bool
 
-- name: "Set fact about Instio Ingress Gateway External IP"
+- name: Set fact about Istio Ingress Gateway External IP
   set_fact:
     istio_external_ip: "{{ istio_web_service.resources[0].status.loadBalancer.ingress[0].ip }}"
   when: seldon_core_analytics_istio_external_ip_required | bool
 
+- name: Set fact about external address when dns not enabled
+  set_fact:
+    external_address: "{{ istio_web_service.resources[0].status.loadBalancer.ingress[0].ip }}"
+  when: not configure_dns | bool
+
+- name: Set fact about external address when dns enabled
+  set_fact:
+    external_address: "{{seldon_deploy_cluster_name}}.cloud.seldon.io"
```

Comment:
This is very use-case specific?

Reply:
Discussed offline, will merge and open a ticket to reconsider the external_address/istio_external_ip set_fact steps.